core/runtime/src/main/java/org/eclipse/sw360/antenna/workflow/stubs/AbstractComplianceChecker.java

```diff
@@ -53,14 +54,22 @@ public AbstractComplianceChecker() {
         try {
             execute(evaluation);
         } catch (FailCausingException e) {
-            // TODO: A FailCausingException should not end the build but should be reflected in the workflow result
-            // Build should not be aborted, but should indicate the failure in the end, so that reports etc. are created.
-            throw new ExecutionException("Fail execution because of compliance check result", e);
+            // no-op
```

Comment:
The FailCausingException was introduced to only support this case here, you can remove it now, that an fail cause is not an execption anymore, this would help in avoiding an empty catch block.